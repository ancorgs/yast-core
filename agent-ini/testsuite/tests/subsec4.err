[agent-ini] IniParser.cc(parse_helper):XXX 5: key2: values at the top level not allowed.
[agent-ini] IniParser.cc(parse_helper):XXX 6: Nothing to close.
