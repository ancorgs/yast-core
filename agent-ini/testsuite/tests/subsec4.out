(["xxx"])
start xxx
sec zzz
key1=val1
secend
end
